### PR TITLE
uftrace(dynamic function graph tracer): bump version

### PR DESCRIPTION
**Pull request by bernhardkaindl**
I am the maintainer and creator of the package in spack:
- This is just a small version update with many little improvements.

### CODE REVIEW

**Comment by bernhardkaindl** [comment]
@spackbot run pipeline

File: var/spack/repos/builtin/packages/uftrace/package.py [hunk 1]
```diff
@@ -37,6 +38,12 @@ class Uftrace(AutotoolsPackage):
     depends_on('python@3.5:', when='+python3')
     depends_on('lua-luajit')
 
+    # Fix the version string if building below another git repo. Submitted to upstream:
+    when('@:0.11')
```
Comment (haampie) [inline var/spack/repos/builtin/packages/uftrace/package.py]: Did you mean to add a decorator?

Comment (bernhardkaindl) [reply var/spack/repos/builtin/packages/uftrace/package.py]: flake8 required me to add the empty line. It's ok, no problem, I wouldn't want to start a new pipeline for that.

Comment (alalazo) [reply var/spack/repos/builtin/packages/uftrace/package.py]: @bernhardkaindl I think this should be:
```
@when('@:0.11')
def patch(self):
    ...
```
if the intent is to have the function run only on versions not greater than 0.11

**Review by alalazo — CHANGES_REQUESTED** [review]

**Comment by bernhardkaindl** [comment]
@alalazo This was the original intention, yes:
```
@when('@:0.11')
def patch(self):
    ...
```
But flake8 does tells me I must add a blank line - with a fresh bootstrap of flake8:
```
==> py-flake8: Successfully installed py-flake8-3.8.2-abd3jsshhu5vtuwyqchrwb3kmazgyaeq
  Fetch: 0.75s.  Build: 1.03s.  Total: 1.78s.
[+] /home/kaindlb/.spack/bootstrap/store/linux-ubuntu18.04-zen/gcc-8.4.0/py-flake8-3.8.2-abd3jsshhu5vtuwyqchrwb3kmazgyaeq
  var/spack/repos/builtin/packages/uftrace/package.py:42: [E301] expected 1 blank line, found 0
  flake8 found errors
```
It is Ok to run the function intented for it always. I could remove it, but did not want to re-trigger the pipelines for that.
You need it removed?

**Comment by alalazo** [comment]
I think `flake8` is complaining due to the missing `@` character in front of the decorator (so it's seeing this as a regular function call).

**Comment by bernhardkaindl** [comment]
> I think `flake8` is complaining due to the missing `@` character in front of the decorator (so it's seeing this as a regular function call).

Thanks, fixed and pushed! (sigh!)

**Review by alalazo — APPROVED** [review]

**Pull request merged**